tests/testthat/_snaps/table_egt03.md

```diff
@@ -3,99 +3,102 @@
+      ————————————————————————————————————————————————————————————————
+      Placebo (N=336)                                                 
+        n                                333          0          3    
+        LOW                          83 (24.7%)       0       1 (0.3%)
+        NORMAL                       243 (72.3%)      0       2 (0.6%)
+        HIGH                          5 (1.5%)        0          0    
+        Missing                       2 (0.6%)        0          0    
+      Xanomeline High Dose (N=324)                                    
+        n                                321          0          3    
+        LOW                          80 (24.7%)       0       1 (0.3%)
+        NORMAL                       241 (74.4%)      0       2 (0.6%)
+        HIGH                              0           0          0    
+        Missing                           0           0          0    
+      Xanomeline Low Dose (N=336)                                     
+        n                                329          1          6    
+        LOW                          83 (24.7%)       0       1 (0.3%)
+        NORMAL                        242 (72%)    1 (0.3%)   5 (1.5%)
+        HIGH                          1 (0.3%)        0          0    
+        Missing                       3 (0.9%)        0          0    
 
 # EGT03 variant 3 is produced correctly
 
     Code
       res
     Output
-                                  LOW         NORMAL     HIGH   Missing 
-      ——————————————————————————————————————————————————————————————————
-      A: Drug X (N=134)                                                 
-        n                          39           93        0        2    
-        LOW                     1 (0.7%)     5 (3.7%)     0        0    
-        NORMAL                 36 (26.9%)   84 (62.7%)    0     1 (0.7%)
-        HIGH                    2 (1.5%)      4 (3%)      0     1 (0.7%)
-      B: Placebo (N=134)                                                
-        n                          42           90        0        2    
-        LOW                     1 (0.7%)     9 (6.7%)     0        0    
-        NORMAL                 40 (29.9%)   77 (57.5%)    0     2 (1.5%)
-        HIGH                    1 (0.7%)      4 (3%)      0        0    
-      C: Combination (N=132)                                            
-        n                          36           95        0        1    
-        LOW                      4 (3%)     11 (8.3%)     0        0    
-        NORMAL                 31 (23.5%)   75 (56.8%)    0     1 (0.8%)
-        HIGH                    1 (0.8%)     9 (6.8%)     0        0    
+                                       LOW        NORMAL        HIGH     Missing 
+      ———————————————————————————————————————————————————————————————————————————
+      Placebo (N=336)                                                            
```

Comment:
double check this number, it seems high? why not 86

Reply:
I have updated these values - the tables were not using ADSL as `alt_counts_df` and so these were the N's coming from ADEG